x-pack/plugins/cases/server/client/utils.ts

```diff
@@ -114,6 +115,25 @@ export const addStatusFilter = ({
   return filters.length > 1 ? nodeBuilder.and(filters) : filters[0];
 };
 
+export const addSeverityFilter = ({
```

Comment:
The `addStatusFilter` and `addSeverityFilter` are almost identical. Could you please combine them into one generic function?

Reply:
After an off-line discussion I decided to keep them separate.